src/azure-cli-core/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.0.22
+++++++
+* Modified the AZURE_US_GOV_CLOUD's AAD authority endpoint from login.microsoftonline.com
```

Comment:
Please make it a single line
Also, please update the [setup.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L18) under the same directory to have the same version

Reply:
Ok thanks, I've reduced the HISTORY.RST change to a single line and updated the version in setup.py. Let me know what else you guys need. Thx!